Makefile.win

```diff
@@ -0,0 +1,228 @@
+-include Makefile.local # for optional local options e.g. threads

```

Comment:
Should this be `Makefile.win.local` to avoid conflicts? The same local directory with local overrides can be used for both Windows and POSIX builds (either via WSL or VMs).